roles/tpa_single_node/tasks/collectorist/api.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Create configmap for collectorist-api
+  ansible.builtin.template:
+    src: "{{ role_path }}/templates/manifests/collectorist/api/Configmap.yaml.j2"
```

Comment:
@i386x this must be changed into `Configmap.yaml` or you have to rename the file from `collectorist/api/Configmap.yaml` to `collectorist/api/Configmap.yaml.j2` because the file isn't found

Reply:
@desmax74 Renamed